gym/utils/seeding.py

```diff
@@ -23,6 +23,12 @@ def np_random(seed: Optional[int] = None) -> Tuple["RandomNumberGenerator", Any]
 # TODO: Remove this class and make it alias to `Generator` in a future Gym release
 # RandomNumberGenerator = np.random.Generator
 class RandomNumberGenerator(np.random.Generator):
+    def __eq__(self, other):
```

Comment:
Why do we need this? Both in the sense of where is it used, and why can't we just use the inherited `__eq__`?

Reply:
Removed in 41a568300a7d91a03e529aaf03ea24e5699bdd4f